### ISSUE TITLE
Grid аdditional `Tab` key required

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#4801]
### Description

With the recent Chrome updates, now scrollable elements are focused. 

https://statics.teams.cdn.office.net/evergreen-assets/safelinks/2/atp-safelinks.html

This requires double pressing the `Tab` key to move the focus from cell to pager.

### Steps To Reproduce

Try to `Tab` through the Grid. Notice how the focus from the first cell isn't moved after the fist `Tab`.
https://demos.telerik.com/kendo-angular-ui/demos/grid/a11y/basic-usage?theme=default-ocean-blue

### Actual Behavior

The focused element isn't changed when clicking `Tab`.

### Expected Behavior

The focus should be moved when `Tab` key is pressed.

### Workaround

Set the `tabindex` of the `k-grid-content` element to -1:

```ts
@ViewChild(GridComponent, { read: ElementRef }) grid!: ElementRef;

  ngAfterViewInit() {
    const gridContent =
      this.grid.nativeElement.querySelector('.k-grid-content');
    if (gridContent) {
      gridContent.setAttribute('tabindex', '-1');
    }
  }
```
